### PR TITLE
Integrations Pagination

### PR DESCRIPTION
**Pull request by BrandonRomano**
[🔍 Preview](https://dev-portal-git-brintegrations-pagination-2-hashicorp.vercel.app/vault/integrations)

### CODE REVIEW

File: src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+		(a: Integration, b: Integration): number => {
+			if (a.name < b.name) {
+				return -1
+			} else if (a.name > b.name) {
+				return 1
+			} else {
+				return 0
+			}
+		}
+	)
```
Comment (thiskevinwang) [inline src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx:29]: Does the hardcoded alphabetical sorting prevent us from doing other sorting methods? — i.e. by "created date", by "most favorited", etc.?

Either way, no action needed, just asking for clarity, and to maybe leave a comment for the intent/expected limitations here.

File: src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+			)}
+		</div>
+	)
+}
+
+// =======================
+
+const COLLAPSED = '...'
+
+function paginatedArray(
```
Comment (thiskevinwang) [inline src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx:67]: Like `unflattenVariables`, i probably will gloss over this and trust that it works :D 

A quick top-level comment for expected output could be useful here 🙏

File: src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx (hunk 1 above)
Comment (BrandonRomano) [reply src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx:29]: It does, but this is the desired sort for now.

In the event in the future we want to add other sorting methods, we would change this here at this level.

File: src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx (hunk 2 above)
Comment (BrandonRomano) [reply src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx:67]: Good idea; I'll add a comment!

**Pull request merged**